### PR TITLE
Adding my command line toolbox - cltoolbox

### PR DESCRIPTION
**Pull request by timcera**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by timcera** [comment]
Hello @conda-forge/staged-recipes,

Been awhile and wondered if there was any problems that needed fixing?

File: recipes/cltoolbox/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: b36dc0eb77ddd43d9a3939dfcff41ca4ac20f497e006d012a6e75e1a1616dc98
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (dopplershift) [inline recipes/cltoolbox/meta.yaml:19]:
```suggestion
    - python >=3.8
```

File: recipes/cltoolbox/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - cltoolbox
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/cltoolbox/
+  summary: Python library for creating and manipulating CLIs
+  license: BSD-3-Clause
```
Comment (dopplershift) [inline recipes/cltoolbox/meta.yaml:37]:
```suggestion
  license: BSD-3-Clause AND MIT
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor things.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**